Os/Linux/File.cpp

```diff
@@ -286,7 +286,7 @@ namespace Os {
                         break;
                     default:
                         DEBUG_PRINT("Error %d during write of 0x0%llx, addrMod %d, size %d, sizeMod %d\n",
-                                    errno, (U64) buffer, ((U64) buffer) % 512, size, size % 512);
+                                    errno, static_cast<POINTER_CAST>(buffer), static_cast<POINTER_CAST>(buffer) % 512, size, size % 512);
```

Comment:
Pointer cast will cause issues with the "ll" flag used here when compiling for systems where the pointer size is not 64bit.